build/build-plugins.js

```diff
@@ -11,12 +11,12 @@
 
 const path = require('path')
 const rollup = require('rollup')
-const glob = require('glob')
+const globby = require('globby')
 const { babel } = require('@rollup/plugin-babel')
 const banner = require('./banner.js')
 
-const srcPath = path.resolve(__dirname, '../js/src/')
-const jsFiles = glob.sync(srcPath + '/**/*.js')
+const srcPath = path.resolve(__dirname, '../js/src/').replace(/\\/g, '/')
```

Comment:
This is needed because globby is using fast-glob: https://github.com/mrmlnc/fast-glob#pattern-syntax